Time_scraper.ipynb

```diff
@@ -0,0 +1,189 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#SCRAPING\n",
+    "import requests\n",
+    "from bs4 import BeautifulSoup\n",
+    "import pandas as pd"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "res = requests.get(\"https://time.com/6171272/elon-musk-twitter-disinformation/\")\n",
```

Comment:
Could you automate this part of the code so that instead of manually setting the URL, you load the list of URL's from google_scraper/results.csv file and then store all parsed results in the DataFrame at the end.